packages/client/components/DiscussionThreadInput.tsx

```diff
@@ -137,13 +148,19 @@ const DiscussionThreadInput = forwardRef((props: Props, ref: any) => {
   const avatar = isAnonymousComment ? anonymousAvatar : picture
   return (
     <Wrapper data-cy={`${dataCy}-wrapper`} ref={ref} isReply={isReply} isDisabled={isDisabled}>
-      <CommentAvatar size={32} picture={avatar} onClick={toggleAnonymous} />
+      <CommentAvatar
+        isEndedMeeting={!!endedAt}
+        size={32}
+        picture={avatar}
+        onClick={toggleAnonymous}
+      />
       <CommentEditor
         dataCy={`${dataCy}`}
         teamId={teamId}
         editorRef={editorRef}
         editorState={editorState}
         onSubmit={onSubmit}
+        readOnly={!!endedAt}
```

Comment:
@ackernaut should discussion threads close after the meeting is complete?

Reply:
If it’s easy enough I’m okay with it. If it’s complicated now or later I’d want to spend more time thinking about the async user stories between sync time, like when folks might pick a deeper discussion back up later, etc.